src/gtk/toga_gtk/widgets/internal/buttons/base.py

```diff
@@ -0,0 +1,66 @@
+class ParentPosition:
+    def __init__(self, adj):
+        self.adj = adj
+
+    def _is_parent_scrollable(self):
```

Comment:
Why are all these methods marked as "private" (i.e., with an underscore prefix)? Effectively, this is a class with no public API - and "public" in this case is "the implementation layer", so there's already no risk of the end-user of Toga being exposed to the class.

Reply:
This is a base class for scrollable rows, it's meant to be inherited from. The inheriting class will only use these methods internally.

Reply:
So... and internal API, that will be subclassed, and the methods *must* be used in the place where it's subclassed. Underscore prefixing is effectively a "you shouldn't be using this method" API - why are we using a prefix here when we explicitly *will* need to use the API?

Reply:
Makes sense.